src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-ai.mdx

```diff
@@ -113,48 +132,140 @@ To chat with New Relic AI about log errors:
 3. Search for `errors`.
 2. Click on any resulting log lines and look for the <DoNotTranslate>**Explain this error**</DoNotTranslate> button at the top of the log details panel. Our AI will provide a summary of the error. 
 
-You can take your investigation further by telling our AI about your stack and asking follow up questions. For example, you can ask: 
+You can take your investigation further by telling our AI about your stack and asking follow-up questions. For example, you can ask: 
 
-* How often is this error happening? Meaning, what's the error rate? Is it 1% or 50%?
-* What hosts are throwing these errors? I would like to know is it across all of my hosts or is it just affecting one or two of them?
+```
+How often does this error occur? What's the error rate? Is it 1% or 50%?
+```
+```
+What hosts are throwing these errors? Is it across all of my hosts, or is it just affecting one or two of them?
+```
 
-### Analyze stack traces in errors inbox [#stack-traces]
+## Analyze stack traces in errors inbox [#stack-traces]
 
-Using our AI to analyze stack traces will reduce context switching and help speed up triage and analysis. 
+Using our AI to analyze stack traces will reduce context switching and help speed up triage and analysis. To investigate stack traces in errors inbox with New Relic AI:
 
 <img
-    title="errors inbox ask ai "
-    alt="A screenshot that shows how to chat with AI in errors inbox"
-    src={aiAskAiinErrorsInbox}
+    title="explain this error"
+    alt="A screenshot showing how to use New Relic AI to investigate stack traces. "
+    src={aiExplainThisErrorStackTraces}
 />
 
 <figcaption>
     To chat with AI in errors inbox, go to <DoNotTranslate>**[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities)**</DoNotTranslate> > <DoNotTranslate>**Errors inbox**</DoNotTranslate> > and select a workload. 
```

Comment:
@homelessbirds since this figcaption is exactly the same as the step below it, how about ditching this figcaption and inserting the image right inside the steps (maybe after step 2). Be sure to indent the image if you do this.

Reply:
That's a good idea, I will do that.